test_unstructured_ingest/test-ingest-pinecone.sh

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env bash
+
+set -e
+
+SCRIPT_DIR=$(dirname "$(realpath "$0")")
+cd "$SCRIPT_DIR"/.. || exit 1
+OUTPUT_FOLDER_NAME=s3-pinecone-dest
+OUTPUT_DIR=$SCRIPT_DIR/structured-output/$OUTPUT_FOLDER_NAME
+WORK_DIR=$SCRIPT_DIR/workdir/$OUTPUT_FOLDER_NAME
+DOWNLOAD_DIR=$SCRIPT_DIR/download/$OUTPUT_FOLDER_NAME
+max_processes=${MAX_PROCESSES:=$(python3 -c "import os; print(os.cpu_count())")}
+
+if [ -z "$OPENAI_API_KEY" ] && [ -z "$PINECONE_API_KEY" ]; then
```

Comment:
why openai? did you prove that we could use huggingface over [here](https://github.com/Unstructured-IO/unstructured/pull/1918)?

Reply:
After that one goes in, we can use it 👍 (it adds support for other embedding modules) 

I'll update accordingly

Reply:
This will be nice!

Reply:
Addressed with [ingest test uses huggingface embedder](https://github.com/Unstructured-IO/unstructured/pull/1774/commits/791bf03d00cfca5a0b6e74066b86f5af7c90ad4e) 👍